packages/clerk-js/src/core/resources/Organization.ts

```diff
@@ -24,12 +26,12 @@ export class Organization extends BaseResource implements OrganizationResource {
     this.fromJSON(data);
   }
 
-  static async create(name: string): Promise<OrganizationResource> {
+  static async create({ name, slug }: CreateOrganizationParams): Promise<OrganizationResource> {
```

Comment:
❓ I'm not entirely familiar with the organisations API... is this a public method intended to be used by the users? If yes, then this would be a breaking change and should probably be handled using overloads. If no, please ignore :)

Reply:
I saw that this is only used by `clerk.createOrganization` but was reluctant to make the change as well.

@igneel64 What's your take regarding backwards compatibility here?

Reply:
We can overload it for now and teach it to accept a string (name) or an object. 

For future reference, **we must always use objects for resource methods even if we have just one parameter**

Reply:
The change would be considered breaking but if we go with the suggestion of:

> We can overload it for now and teach it to accept a string (name) or an object.

I think we are good.

Reply:
If we go with overloads, please add a deprecation warning for the non-object form so we can remove it in v4

Reply:
Could you please take a look at this commit? https://github.com/clerkinc/javascript/pull/191/commits/0fea8df722f99673c20ba8b48952c4574dc9c501

Is this what you guys had in mind regarding deprecation?